src/Api/Api.php

```diff
@@ -58,6 +60,8 @@ public function getImageManager()
      * Set the manipulators.
      *
      * @param array $manipulators Collection of manipulators.
+     *
+     * @return void
```

Comment:
Can't we start using real return types now, since the minimal PHP requirement is 7.2?

Reply:
Personally I would love to add real param and return types :) Hopefully we can do so soon.

Reply:
Let's add psalm in the minor and types in the next major IMO